app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = [Person(people[i]["name"], people[i]["age"])
+               for i in range(len(people))]
```

Comment:
For future reference, the correct plural form is `people`. You should call it `people_list`, for  example